tests/attestation_plugin_cert/host/host.cpp

```diff
@@ -119,24 +121,43 @@ void run_test(oe_enclave_t* enclave, int test_type)
 #else
         file = fopen(filename, "wb");
 #endif
-        fwrite(cert, 1, cert_size, file);
+        fwrite(certificate, 1, certificate_size, file);
         fclose(file);
     }
 
-    // validate cert
+    // validate certificate
     OE_TRACE_INFO("Host: Verifying tls certificate\n");
-    OE_TRACE_INFO("Host: cert = %p cert_size = %d\n", cert, cert_size);
-    result = oe_verify_attestation_certificate_with_evidence(
-        cert, cert_size, enclave_claims_verifier, nullptr);
+    OE_TRACE_INFO(
+        "Host: certificate = %p certificate_size = %d\n",
+        certificate,
+        certificate_size);
+    result = oe_verify_attestation_certificate_with_evidence_v2(
+        certificate,
+        certificate_size,
+        nullptr,
+        0,
+        nullptr,
+        0,
+        &claims,
+        &claims_length);
+
     OE_TRACE_INFO(
         "Host: Verifying the certificate from a host ... %s\n",
         result == OE_OK ? "Success" : "Fail");
+
+    result = enclave_claims_verifier(claims, claims_length, nullptr);
```

Comment:
Similiar comment above, consider adding OE_TEST(result == OE_OK); here.
And since `enclave_claims_verifier` is no longer a callback with v2 API, update the comment at Line 27.